src/modules/app/route-configs/TunewsRouteConfig.js

```diff
@@ -29,7 +32,11 @@ const tunewsRoute: Route = {
       fetchData(createCitiesEndpoint(cmsApiBaseUrl), dispatch, state.cities),
       fetchData(createEventsEndpoint(cmsApiBaseUrl), dispatch, state.events, { city, language }),
       fetchData(createTunewsLanguagesEndpoint(tunewsApiBaseUrl, true), dispatch, state.languages, { city, language }),
-      fetchData(createTunewsEndpoint(tunewsApiBaseUrl), dispatch, state.tunews, { page: 1, language: language, count: 20 })
+      fetchData(createTunewsEndpoint(tunewsApiBaseUrl), dispatch, state.tunews, {
```

Comment:
Try to avoid these changes as no linting rule was violated. If the line was too long for the linter then this is fine though :+1:

Reply:
Ok. Just hit autoformat I guess...

Reply:
But I guess the line was too long, so it should be fine, right?